### PR TITLE
Remove govuk_frontend_toolkit sass dependency

### PR DESCRIPTION
**Pull request by andysellick**
Remove toolkit sass dependency. Should be no visual changes.

(some) Pages to check:

- https://government-frontend-pr-1529.herokuapp.com
- https://government-frontend-pr-1529.herokuapp.com/government/organisations/attorney-generals-office/about
- https://government-frontend-pr-1529.herokuapp.com/government/case-studies/get-britain-building-carlisle-park
- https://government-frontend-pr-1529.herokuapp.com/government/publications/car-show-me-tell-me-vehicle-safety-questions/car-show-me-tell-me-vehicle-safety-questions
- https://government-frontend-pr-1529.herokuapp.com/log-in-file-self-assessment-tax-return

---

Component guide for this PR:
https://government-frontend-pr-1529.herokuapp.com/component-guide

### CODE REVIEW

File: app/assets/stylesheets/components/_error-message.scss [hunk 1]
```diff
@@ -2,12 +2,8 @@
 
 .app-c-error-message {
   display: block;
-
   margin-bottom: $app-spacing-scale-3;
-
   clear: both;
-
   color: $app-error-colour;
```
Comment (vanitabarrett) [inline app/assets/stylesheets/components/_error-message.scss]: Can we use `$govuk-error-colour`? It looks like `$app-error-colour` is defined [here](https://github.com/alphagov/government-frontend/blob/master/app/assets/stylesheets/components/helpers/_variables.scss#L24) as being equal to `$red`, which I'm going to guess is a colour from toolkit.

File: app/assets/stylesheets/components/helpers/_variables.scss [hunk 2]
```diff
@@ -10,8 +10,8 @@ $app-spacing-scale-6: 40px;
 $app-spacing-scale-7: 50px;
 $app-spacing-scale-8: 60px;
 
-$app-text-colour: $text-colour;
-$app-secondary-text-colour: $grey-1;
+$app-text-colour: $govuk-text-colour;
+$app-secondary-text-colour: govuk-colour("dark-grey", $legacy: "grey-1");
 
 // Border widths
 $app-border-width-mobile: 4px;
```
Comment (vanitabarrett) [inline app/assets/stylesheets/components/helpers/_variables.scss]: Line 24: `$app-error-colour: $red;` - does "$red" come from toolkit? I think we can remove this line and update where `$app-error-colour` is used (as mentioned in a previous comment)

File: app/assets/stylesheets/components/helpers/_variables.scss [hunk 3]
```diff
@@ -10,8 +10,8 @@ $app-spacing-scale-6: 40px;
 $app-spacing-scale-7: 50px;
```
Comment (vanitabarrett) [inline app/assets/stylesheets/components/helpers/_variables.scss]: Can we remove this spacing scale and replace uses with the govuk-spacing scale, seeing as this was forked from GOVUK  Frontend in the first place? It looks like it's only actually used in one place, so hopefully is easy to replace.

File: app/assets/stylesheets/components/_error-message.scss (hunk 1 above)
Comment (andysellick) [reply app/assets/stylesheets/components/_error-message.scss]: Done, good spot, thanks.

File: app/assets/stylesheets/components/helpers/_variables.scss (hunk 2 above)
Comment (andysellick) [reply app/assets/stylesheets/components/helpers/_variables.scss]: Done, good spot, thanks.

File: app/assets/stylesheets/components/helpers/_variables.scss (hunk 3 above)
Comment (andysellick) [reply app/assets/stylesheets/components/helpers/_variables.scss]: Done, good spot, thanks.

**Pull request merged**